sdk/common/src/main/java/io/opentelemetry/sdk/internal/ComponentRegistry.java

```diff
@@ -22,18 +22,6 @@
 
   private final ConcurrentMap<InstrumentationLibraryInfo, V> registry = new ConcurrentHashMap<>();
 
-  /**
-   * Returns the registered value associated with this name and {@code null} version if any,
-   * otherwise creates a new instance and associates it with the given name and {@code null}
-   * version.
-   *
-   * @param instrumentationName the name of the instrumentation library.
-   * @return the registered value associated with this name and {@code null} version.
-   */
-  public final V get(String instrumentationName) {
```

Comment:
The method was no longer used by code within this repository after I changed how `MeterSdkProvider` and `TracerSdkProvider` worked. Since it is in the `internal` package, I figured it wasn't intended for public consumption, and rather than having the unused method in the repo, I removed it.

If I should leave it, please let me know and I'll happily undo the change to remove it :)

Reply:
@MariusVolkhart I suggest undoing this change and removing it in a follow-up PR ;)

Reply:
I've reverted my changes, so the `ComponentRegistry` is untouched. Thanks @carlosalberto